pkg/client/unversioned/clientcmd/loader.go

```diff
@@ -36,11 +36,12 @@ import (
 const (
 	RecommendedConfigPathFlag   = "kubeconfig"
 	RecommendedConfigPathEnvVar = "KUBECONFIG"
-	RecommendedHomeFileName     = "/.kube/config"
+	RecommendedHomeDir          = ".kube"
+	RecommendedFileName         = "config"
 )
 
 var OldRecommendedHomeFile = path.Join(os.Getenv("HOME"), "/.kube/.kubeconfig")
-var RecommendedHomeFile = path.Join(os.Getenv("HOME"), RecommendedHomeFileName)
+var RecommendedHomeFile = path.Join(os.Getenv("HOME"), RecommendedHomeDir, RecommendedFileName)
```

Comment:
I would suggest adding
```
var RecommendedSchemaFile = path.Join(os.Getenv("HOME"), RecommendedHomeDir, "schema")
```
and using the var below as the `--schema-cache-dir` default value and in the help text.